xbmc/input/Action.cpp

```diff
@@ -11,13 +11,18 @@
 #include "ActionTranslator.h"
 #include "Key.h"
 
+CAction::CAction() :
+  m_id(ACTION_NONE)
```

Comment:
use default member initialiser

Reply:
I wanted to avoid including ActionIDs.h in the header. But I'll just initialize to 0.

Reply:
Keep it, if you wouldn't use `ACTION_NONE`.